src/feature/dashboard/presentation/dashboard.screen.tsx

```diff
@@ -1,10 +1,280 @@
+      setReservedBook(null);
+    } else {
+      setSnackbarMessage('Something went wrong. Try again later.');
+      setIsSnackbarOpen(true);
+      setIsBorrowModalOpen(false);
+      setReservedBook(null);
+    }
+  };
+
   return (
-    <Paperbase>
-      <div>Dashboard</div>
-    </Paperbase>
+    <>
+      <Paperbase>
+        <Paper sx={{ maxWidth: 936, margin: 'auto', overflow: 'hidden' }}>
+          <AppBar
+            position="static"
+            color="default"
+            elevation={0}
+            sx={{ borderBottom: '1px solid rgba(0, 0, 0, 0.12)' }}
+          >
+            <Toolbar>
+              <Grid container spacing={2} alignItems="center">
+                <Grid item>
+                  <SearchIcon color="inherit" sx={{ display: 'block' }} />
+                </Grid>
+                <Grid item xs>
+                  <TextField
+                    fullWidth
+                    placeholder="Search through books by title, author or ISBN"
+                    InputProps={{
+                      disableUnderline: true,
+                      sx: { fontSize: 'default' },
+                    }}
+                    variant="standard"
+                    onChange={(e) => {
+                      setSearchData(e.target.value);
+                    }}
+                    value={searchData}
+                  />
+                </Grid>
+              </Grid>
+            </Toolbar>
+          </AppBar>
+          <Grid container spacing={4} sx={{ p: 6 }}>
+            <TableContainer>
+              <Table>
+                <TableHead>
+                  <TableRow>
+                    <TableCell>
+                      <Typography variant="subtitle2" fontWeight={600}>
+                        Id
+                      </Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Typography variant="subtitle2" fontWeight={600}>
+                        Title
+                      </Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Typography variant="subtitle2" fontWeight={600}>
+                        Author
+                      </Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Typography variant="subtitle2" fontWeight={600}>
+                        Year
+                      </Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Typography variant="subtitle2" fontWeight={600}>
+                        Category
+                      </Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Typography variant="subtitle2" fontWeight={600}>
+                        ISBN
+                      </Typography>
+                    </TableCell>
+                    <TableCell></TableCell>
+                  </TableRow>
+                </TableHead>
+                <TableBody>
```

Comment:
Same as above, but instead of `booksTableHeader.component.tsx`, `booksTableBody.component.tsx`